website/docs/docs/collaborate/explore-projects.md

```diff
@@ -0,0 +1,142 @@
+
+To display the details about your [project state](/docs/dbt-cloud-apis/project-state), dbt Explorer utilizes the metadata provided through the [Discovery API](/docs/dbt-cloud-apis/discovery-api). The metadata that's available on your project depends on the [deployment environment](/docs/deploy/deploy-environments) you've designated as _production_ in your dbt Cloud project. dbt Explorer automatically retrieves the metadata updates after each job run in the production deployment environment so it will always have the latest state on your project. The metadata it displays depends on the [commands executed by the jobs](/docs/deploy/job-commands). For instance:
+
+- To update model details or results, you must run `dbt run` or `dbt build` on a given model within a job in the environment.
+- To view catalog statistics and columns, you must run `dbt docs generate` within a job in the environment.
+- To view test results, you must run `dbt test` or `dbt build` within a job in the environment.
+- To view source freshness check results, you must run `dbt source freshness` within a job in the environment.
+
+The need to run these commands will diminish and richer, more timely metadata will become available as the Discovery API and its underlying platform evolve.
+
+:::tip Join the beta
+
+dbt Explorer is a [beta feature](/docs/dbt-versions/product-lifecycles#dbt-cloud) and subject to change without notification. More updates to this feature coming soon.
+
+If you’re interested in joining the beta, please contact your account team.
+ 
+:::
+
+## Prerequisites
+
+- You have a [multi-tenant](/docs/cloud/about-cloud/tenancy#multi-tenant) or AWS single-tenant dbt Cloud account on the [Team or Enterprise plan](https://www.getdbt.com/pricing/).
+- You have set up a [production deployment environment](/docs/deploy/deploy-environments#set-as-production-environment-beta) for each project you want to explore.
+    - There has been at least one successful job run in the production deployment environment.
+- You are on the dbt Explorer page.
+    - To go to the page, select **Explore (Beta)** from the top navigation bar in dbt Cloud.
+
+## Explore the project’s lineage
+
+dbt Explorer provides a visualization of your project’s <Term id="dag">DAG</Term> that you can interact with. To start, select **Overview** in the left sidebar and click the **Explore Lineage** button on the main (center) section of the page. 
+
+If you don't see the lineage graph immediately, click **Render Lineage**. It can take some time for the graph to render depending on the size of your project and your computer’s available memory. The graph of very large projects might not render so, instead, you can select a subset of nodes by using selectors.
```

Comment:
Should we give a sense of what a "very large project" is or is it all very relative to a lot of things?